nestkernel/simulation_manager.cpp

```diff
@@ -325,16 +325,41 @@ nest::SimulationManager::simulate( Time const& t )
       "Simulate "
       "is called repeatedly with simulation times that are not multiples of the minimal delay." );
 
-  resume_();
+  resume_( num_active_nodes );
 
   finalize_simulation_();
 }
 
 void
-nest::SimulationManager::resume_()
+nest::SimulationManager::resume_( size_t num_active_nodes )
```

Comment:
Is this (by the way const) num_active_nodes commented as simply a logging value and not actually usable to alter the number of active nodes? Or should this be a SimulationManager member?